### DISCUSSION TITLE
求推荐 支持像设计原型一样 设计前端页面、导出前端代码的低代码平台/代码生成器

### DISCUSSION THREAD

**Discussion by ghost** [taowen/awesome-lowcode#125] [Q&A]
求推荐 支持像设计原型一样 设计前端页面、导出前端代码的低代码平台/代码生成器

想让产品直接做好前端页面，然后导出源码给前端
跳过画原型的步骤

有没有类似iVX这样的，设计功能很强，而且可以导出没有混淆过的源码？

**Comment by channely** [comment]
目前在调研这个，[Quant UX](https://quant-ux.com/)，模式并不是导出源码给前端用（这个方式并不理想），而是结合一个叫Luisa的组件库直接导出业务界面配置文件给前端用。基本能将 视觉、交互、数据绑定、低代码等角色协同到一起。
